ai/categories/reference.md

```diff
@@ -1,3064 +1,3058 @@
+            }
+          ],
+          local_xcm: undefined,
+          destination: {
+            type: 'V4',
+            value: {
+              parents: 0,
+              interior: { type: 'X1', value: { type: 'Parachain', value: 4001 } }
+            }
+          },
+          remote_xcm: {
+            type: 'V3',
+            value: [
+              {
+                type: 'ReserveAssetDeposited',
+                value: [
+                  {
+                    id: {
+                      type: 'Concrete',
+                      value: {
+                        parents: 1,
+                        interior: { type: 'Here', value: undefined }
+                      }
+                    },
+                    fun: { type: 'Fungible', value: 120000000000n }
+                  }
+                ]
+              },
+              { type: 'ClearOrigin', value: undefined },
+              {
+                type: 'BuyExecution',
+                value: {
+                  fees: {
+                    id: {
+                      type: 'Concrete',
+                      value: {
+                        parents: 1,
+                        interior: { type: 'Here', value: undefined }
+                      }
+                    },
+                    fun: { type: 'Fungible', value: 120000000000n }
+                  },
+                  weight_limit: { type: 'Unlimited', value: undefined }
+                }
+              },
+              {
+                type: 'DepositAsset',
+                value: {
+                  assets: { type: 'Wild', value: { type: 'AllCounted', value: 1 } },
+                  beneficiary: {
+                    parents: 0,
+                    interior: {
+                      type: 'X1',
+                      value: {
+                        type: 'AccountId32',
+                        value: {
+                          network: undefined,
+                          id: FixedSizeBinary {
+                            asText: [Function (anonymous)],
+                            asHex: [Function (anonymous)],
+                            asOpaqueHex: [Function (anonymous)],
+                            asBytes: [Function (anonymous)],
+                            asOpaqueBytes: [Function (anonymous)]
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              {
+                type: 'SetTopic',
+                value: FixedSizeBinary {
+                  asText: [Function (anonymous)],
+                  asHex: [Function (anonymous)],
+                  asOpaqueHex: [Function (anonymous)],
+                  asBytes: [Function (anonymous)],
+                  asOpaqueBytes: [Function (anonymous)]
+                }
+              }
+            ]
+          }
+        }      
+      </pre>
+    </div>
+
+                ...
+    <div id="termynal" data-termynal>
+      <pre>
```

Comment:
For some reason this terminal output is being duplicated? I compared to what's on the actual site because this is a long output and wasn't sure, but there's only one terminal element on the associated page: http://127.0.0.1:8000/develop/interoperability/xcm-runtime-apis/#dry-run-api